templates/results-table.php

```diff
@@ -0,0 +1,32 @@
+<script type="text/template" id="tmpl-plugin-check-results-table">
+	<table id="plugin-check__results-table-{{data.index}}" class="plugin-check__results-table">
+		<thead>
+			<tr>
+				<td>
+					FILE:
+				</td>
+				<td colspan="4">
+					{{ data.file }}
+				</td>
+			</tr>
```

Comment:
I am not sure that this makes sense as a table header. Maybe H tag makes more tense here.